predictors_reliability.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<p>I would chance the column name to something like: <code>pc_1, pc_2</code>  instead of <code>principal component 1</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/santiviquez/lie-detection/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>